src/main/java/com/google/sps/servlets/ReasonServlet.java

```diff
@@ -0,0 +1,30 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.Arrays;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that sends reason for use to /reason. (will change to write to database)*/
+@WebServlet("/reason")
+public final class ReasonServlet extends HttpServlet {
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    long timestamp = System.currentTimeMillis();
```

Comment:
On my website I had a line like this and G suggested I use Instant instead. https://docs.oracle.com/javase/tutorial/datetime/iso/instant.html

You can use Instant.now() to get a timestamp of the current time. If you call instant.toString() it gives a nice timestamp representation of the time.

Reply:
Ok thank you! This seems very helpful!